cli/src/cmd.ts

```diff
@@ -1,64 +0,0 @@
-import {
-  type Network,
-  networks,
-  defaultPanopticfactoryAddress,
-  defaultUniswapFactoryAddress,
-  defaultWalletPrivateKey,
-  defaultPanopticHelperAddress
-} from './config.js'
-import yargs from 'yargs/yargs'
-import { hideBin } from 'yargs/helpers'
-import process from 'process'
-import options from './options.js'
-import { type Address, type Hex } from 'viem'
-
-export const cmd = await yargs(hideBin(process.argv))
-  // .command('lp', 'Show liquidity provider tools')
-  .command('start', 'Enter interactive mode')
-  .help(
-    'Interactively communicate with a Panoptic option pool deployment. Configuration parameters (network, contract address, wallet...) can be set using command line arguments, or inside the interactive interface'
-  )
-  .options(options)
-  .strict()
-  // .demandCommand()
-  .showHelp().argv
-
-export const isCommand = (command: string) => cmd._[0] === command
-
-export const parseInitialNetwork = (): Network => {
```

Comment:
Related to another comment about processing options outside of React - this function (as well as stuff below) was put here instead of inside the `.tsx` components was for the same reason - basically doing as much as possible outside of React's context to avoid impact from redundant processing / conflicts / errors from React

Reply:
It would also save a lot of code modification (e.g.  the conundrums inside Client components - removing `setNetwork`, having to pull client setup inside the React components because options processing are inside React hooks thus not accessible outside a component, etc.)

Reply:
Resolved now